### PR TITLE
Add TSC meeting notes for 01-25-2021

### PR DESCRIPTION
**Pull request by michdolan**
Signed-off-by: Michael Dolan <michdolan@gmail.com>

### CODE REVIEW

File: ASWF/meetings/tsc/2021-01-25.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    - Patrick: Some apps make sense to keep with core, but strange for some to 
+      be out of repo.
+    - Sean: Good to distinguish between OCIO apps and utilities. Utils include 
+      ocioperf, ociocheck, etc. Apps like ociodisplay, ocioconvert, etc.
+    - Patrick: ociodisplay more of an integration example.
+    - Sean: ociodisplay and ocioconvert are useful for debugging CPU/GPU 
+      renderer and finding cause of artifacts.
+    - Michael: If we split within repo, do we have two builds or just an apps 
+      option, like we have now but defulting to OFF.
+    - Mark B: CMake doesn't do well with finding internal depednencies. Won't
```
Comment (hodoulp) [inline ASWF/meetings/tsc/2021-01-25.md:38]: typo dependencies

File: ASWF/meetings/tsc/2021-01-25.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+      renderer and finding cause of artifacts.
+    - Michael: If we split within repo, do we have two builds or just an apps 
+      option, like we have now but defulting to OFF.
+    - Mark B: CMake doesn't do well with finding internal depednencies. Won't 
+      see the installed version of lib. Will try to rebuild and link against 
+      it. Benefit of separate repo is no cross-dependency between app and lib. 
+      Could be in one repo, and change dependencies of apps to depend on new 
+      installed target. Needs to be done in two steps.
+    - Patrick: Becomes an additional process for development.
+    - Mark B: Could write Cmake modue to find lib if it exists locally.
```
Comment (hodoulp) [inline ASWF/meetings/tsc/2021-01-25.md:44]: typo -> 'module'

**Review by hodoulp — APPROVED** [review]

**Review by doug-walker — APPROVED** [review]

File: ASWF/meetings/tsc/2021-01-25.md (hunk 1 above)
Comment (michdolan) [reply ASWF/meetings/tsc/2021-01-25.md]: Fixed

File: ASWF/meetings/tsc/2021-01-25.md (hunk 2 above)
Comment (michdolan) [reply ASWF/meetings/tsc/2021-01-25.md]: Fixed

**Review by hodoulp — APPROVED** [review]

**Comment by michdolan** [comment]
Closing PR since these notes have been moved to Confluence: https://wiki.aswf.io/display/OCIO/2021-01-25

**Pull request closed without merging**